docs/tutorial.md

```diff
@@ -0,0 +1,71 @@
+# Tutorial: First 15 Minutes
+This tutorial walks you through setting up Tilt for your project. It should take 15 minutes, and assumes you've already [installed Tilt](install.html). Before you begin, you may want to join the `#tilt` channel in [Kubernetes Slack](http://slack.k8s.io) for technical and moral support.
+
+Start by `cd`'ing into a project you can already build and deploy.
+
+## Example Tiltfile
+At the end of this guide, your Tiltfile will look something like this:
+```python
+# Deploy: tell Tilt what yaml to deploy
+k8s_yaml('app.yaml')
+
+# Build: tell Tilt what images to build from which directories
+docker_build('companyname/frontend', 'frontend')
+docker_build('companyname/backend', 'backend')
+# ...
+
+# Watch: tell Tilt how to connect locally (optional)
+k8s_resource('frontend', port_forwards=8080)
+```
+
+## Hello World
+Run `tilt up` to enter Tilt's Heads-Up Display. Instead of writing your configuration all at once, we'll use Tilt interactively. Each time you save your configuration, Tilt will reexecute it. Tilt should be complaining there's no file named `Tiltfile`. Open it in your editor and write:
+```python
+print('Hello Tiltfile')
+```
+
+Now save the file. Congrats, you just ran your first `Tiltfile`. Tilt's configurations are programs in [Starlark](https://github.com/bazelbuild/starlark#tour>), a dialect of Python. Can you see "Hello Tiltfile" in Tilt's UI? Tilt is also warning you there are no declared resources. Let's add some.
+
+## Step 1: Deploy
+The function `k8s_yaml` registers Kubernetes objects you want to deploy:
+```python
+k8s_yaml('app.yaml')
+```
+
+Tilt supports many deployment configuration practices (for more details, check out the [Deploy](tiltfile_concepts.html#deploy) section of "Tiltfile Concepts"):
+```python
+# multiple yaml files; can be either a list or multiple calls
+k8s_yaml(['foo.yaml', 'bar.yaml'])
+
+# run a command to generate yaml
+k8s_yaml(local('gen_k8s_yaml.py')) # a custom script
+k8s_yaml(kustomize('config_dir')) # built-in support for popular tools
+k8s_yaml(helm('chart_dir'))
+```
+
+Use the pattern that matches your project (if you're not sure, feel free to ask in [Slack](index.html#community)). You can see when it works because Tilt will display the registered objects.
```

Comment:
(Can punt for a future PR but) might be worth putting in screenshots here and elsewhere along the way?

Reply:
Yeah, screenshots would be good.